functions/Set-DbaTempDbConfig.ps1

```diff
@@ -203,13 +203,21 @@ function Set-DbaTempDbConfig {
 
             $logfile = Get-DbaDbFile -SqlInstance $server -Database tempdb | Where-Object Type -eq 1 | Select-Object LogicalName, PhysicalName, @{L = "SizeMb"; E = { $_.Size.Megabyte } }
 
-            if ($LogFileSize) {
+            if ($LogPath -or $LogFileSize) {
                 $Filename = Split-Path $logfile.PhysicalName -Leaf
                 $LogicalName = $logfile.LogicalName
-                $NewPath = "$LogPath\$Filename"
+                
+                if ($logPath) {
```

Comment:
logPath -> LogPath (sorry, I like the correct case)

Reply:
Sorry! Absolutely :)